canonical-web-and-design/ubuntu.com
# Remove and redirect /download/orange-pi page

## Done

- Remove and redirect /download/orange-pi page and all the older redirects
- Archived the copy doc

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/download/orange-pi-zero
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page is removed and redirected to the [orange-pi download site](http://www.orangepi.org/downloadresources/)


## Issue / Card

Fixes #7868
